test/test_grudge.py

```diff
@@ -850,7 +850,7 @@ def analytic_sol(x, t=0):
         def rhs(t, w):
             return maxwell_operator.operator(t, w)
 
-        dt = 0.002
```

Comment:
Could you add some tests that verify the scaling properties of the `dt_*_factor` routines? (probably best in a separate file)

* Increasing the order superlinearly decreases the non-geometric factor.
* Uniform scaling of the mesh reduces the geometric factor by exactly the scale factor.
* Single-axis scaling of the mesh reduces the geometric factor by a factor related to the scale factor

Reply:
@inducer I've added a refinement test for the geometric scaling factors, but I'm not quite sure how to craft a test for the non-geometric factors. Got any hints?

Reply:
I was thinking vary the order between like 1 and 8 and check `lower_bound <= nongeom * n**2 <= upper_bound`. Kinda crude, but better than nothing.

Reply:
Thanks! I think I got a decent test set up here: https://github.com/inducer/grudge/pull/108/commits/267fa878f7b3b93d5b5fe392480a5cd202cd9f9d